_episodes/interviews/2022-12-02-gray-olson.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Presser with Gray Olson"
+date: 2022-12-02T17:13:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/2022-12-02-gray-olson.mp3
+duration: "1:11:15"
+length: "68409887"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+Allen Wyma talks with [Gray Olson](https://www.grayolson.me/), developer of [Presser](https://github.com/EmbarkStudios/presser), a library that helps with copying data into raw buffers.
```

Comment:
What are "raw buffers" here? Can we be more specific? It sounds like it has to do with graphics somehow?